tools/ui-components/utils/gen-component-template.ts

```diff
@@ -4,28 +4,32 @@ import React from 'react';
 
 import { ${name}Props } from './types';
 
-export const ${name} = ({}: ${name}Props) => {
-  return <div>Hello, I am a ${name} component</div>;
+export const ${name} = (props: ${name}Props):JSX.Element => {
+  return <div {...props}>Hello, I am a ${name} component</div>;
 };
 `;
 
 // types.ts
 export const type = (name: string): string => `
 export interface ${name}Props {
-  className?: string
+  className?: string;
+  children?: string;
```

Comment:
Yeah, it should be similar to formcontrol component, because it's an input 🤔 and inputs should not have children.

But I think spacer should accept children maybe we want to include ` <hr />`in it.

Anyway I see your point I will remove it and think of a way around the test